todo/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "todo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-redux": "^7.0.3",
```

Comment:
Make sure to do `yarn add styled-components` too, since you are using styled components in this project.